packages/generator/test/prisma/field.test.ts

```diff
@@ -62,21 +60,21 @@ describe("Field model", () => {
 
   it("handles hasOne relations", () => {
     expect(Field.parse("hasOne:task").toString()).toMatchInlineSnapshot(`"task  Task"`)
-    expect(Field.parse("hasOne:tasks").toString()).toMatchInlineSnapshot(`"task  Task"`)
+    expect(Field.parse("hasOne:tasks").toString()).toMatchInlineSnapshot(`"tasks  Task"`)
     expect(Field.parse("hasOne:task?").toString()).toMatchInlineSnapshot(`"task  Task?"`)
-    expect(Field.parse("hasOne:tasks?").toString()).toMatchInlineSnapshot(`"task  Task?"`)
+    expect(Field.parse("hasOne:tasks?").toString()).toMatchInlineSnapshot(`"tasks  Task?"`)
     // list identifier should be ignored for singular relations
     expect(Field.parse("hasOne:task[]").toString()).toMatchInlineSnapshot(`"task  Task"`)
-    expect(Field.parse("hasOne:tasks[]").toString()).toMatchInlineSnapshot(`"task  Task"`)
+    expect(Field.parse("hasOne:tasks[]").toString()).toMatchInlineSnapshot(`"tasks  Task"`)
   })
 
   it("handles hasMany relations", () => {
-    expect(Field.parse("hasMany:task").toString()).toMatchInlineSnapshot(`"tasks  Task[]"`)
```

Comment:
Prevent field name from being converted to plural forms when creating a Field with a list type property.